package.json

```diff
@@ -2,18 +2,18 @@
   "name": "cachet",
   "devDependencies": {
     "gulp": "~3.9.1",
-    "laravel-elixir": "~5.0.0",
+    "laravel-elixir": "~6.0.0-9",
     "animate-sass": "git+https://github.com/tgdev/animate-sass.git",
     "autosize": "^3.0.15",
     "bootstrap-sass": "^3.3.6",
     "chart.js": "^2.1.2",
-    "eonasdan-bootstrap-datetimepicker": "~3.1",
+    "eonasdan-bootstrap-datetimepicker": "~4.15",
     "github-markdown-css": "^2.3.0",
     "ionicons": "~2.0",
     "jquery-minicolors": "^2.1.10",
     "jquery-serializeobject": "^1.0.0",
     "jquery-sparkline": "^2.3.2",
-    "jquery": "~2.2",
+    "jquery": "~3.1",
```

Comment:
I'm not sure about upgrading to jQuery v3 yet, customers may not be browsing Cachet on a new browser. Although, what's support like?